src/drivers/nuttx/lv_nuttx_entry.c

```diff
@@ -104,7 +109,9 @@ void lv_nuttx_init(const lv_nuttx_dsc_t * dsc, lv_nuttx_result_t * result)
 
     lv_nuttx_cache_init();
 
+#if LV_CACHE_DEF_SIZE > 0
     lv_nuttx_image_cache_init();
+#endif
```

Comment:
```suggestion
    lv_nuttx_image_cache_init();
```

This is not necessary. LV_CACHE_DEF_SIZE now only serves as a guide, and is only passed into lvgl as a parameter when lv_init. If LV_CACHE_DEF_SIZE starts with 0, it can also be reset to a non-0 value later, and lv_nuttx_image_cache will dynamically increase the size of the image cache.

All this code related to image cache has become runtime dynamic.

Reply:
Thanks you, fixed!